lib/platforms/twilio/twilio.js

```diff
@@ -35,7 +35,7 @@ var Twilio = new ChatExpress({
   routes: {
     '/redbot/twilio': function(req, res) {
       this.receive(req.body);
-      res.send(200);
+      res.send(''); // sending 200 cause an "OK" in whatsup
```

Comment:
I treid to apply this fix locally but that stopped the service from working completely, so not sure if that will fix it

Reply:
@jekkos sorry for the delay, I've tried and it works, how did you configured your twilio account?

Reply:
I'm using whatsapp through the sandbox currenttly, as this API is not yet open completely.

Reply:
So I'm just using the twilio sender/receiver with the sandbox as a sender number.